README.md

```diff
@@ -17,83 +17,93 @@ import UIKit
-        // PreferenceManager.getDefaultSharedPreferences(activity).getString(EU_CONSENT_KEY, null);
-        // consentUUID will be available as String at cLib.consentUUID and under
-        // PreferenceManager.getDefaultSharedPreferences(activity).getString(CONSENT_UUID_KEY null);
-        consentViewController.onInteractionComplete = { cvc in
-            do {
-                print(
-                    "\n eu consent prop",
-                    cvc.euconsent as Any,
-                    "\n consent uuid prop",
-                    cvc.consentUUID as Any,
-                    "\n eu consent in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.EU_CONSENT_KEY) as Any,
-                    "\n consent uuid in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.CONSENT_UUID_KEY) as Any,
-
-                    // Standard IAB values in UserDefaults
-                    "\n IABConsent_ConsentString in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_CONSENT_STRING) as Any,
-                    "\n IABConsent_ParsedPurposeConsents in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_PARSED_PURPOSE_CONSENTS) as Any,
-                    "\n IABConsent_ParsedVendorConsents in storage",
-                    UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_PARSED_VENDOR_CONSENTS) as Any,
-
-                    // API for getting IAB Purpose Consents
-                    "\n IAB purpose consent for \"Ad selection, delivery, reporting\"",
-                    cvc.getIABPurposeConsents([3])
-                )
-                print("Custom vendor consents")
-                for consent in try cvc.getCustomVendorConsents() {
-                    print("Custom Vendor Consent id: \(consent.id), name: \(consent.name)")
-                }
-                print("Custom purpose consents")
-                for consent in try cvc.getCustomPurposeConsents() {
-                    print("Custom Purpose Consent id: \(consent.id), name: \(consent.name)")
-                }
-            }
-            catch { print(error) }
-        }
-
-        view.backgroundColor = UIColor.gray
-
-        view.addSubview(consentViewController.view)
-
-        // IABConsent_CMPPresent must be set immediately after loading the ConsentViewController
-        print(
-            "IABConsent_CMPPresent in storage",
-            UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_CMP_PRESENT) as Any,
-            "IABConsent_SubjectToGDPR in storage",
-            UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_SUBJECT_TO_GDPR) as Any
-        )
-    }
+var consentViewController: ConsentViewController!
+
+private func buildConsentViewController(showPM: Bool, addToView parentView: UIView) {
+do {
```

Comment:
Hi @vilasma can you make sure that the code is correctly indented in the README? I believe the indentation was lost due to copy and paste.